Sources/StreamChatUI/Appearance+ColorPalette.swift

```diff
@@ -16,6 +16,7 @@ public extension Appearance {
         /// Static color which should stay the same in dark and light mode, because it's only used as text on small UI Elements
         /// such as `ChatUnreadCountView`, `GiphyBadge` or Commands icon.
         public var staticColorText: UIColor = .streamWhiteStatic
+        public var staticBlackColorText: UIColor = .streamBlackStatic
```

Comment:
Where do we need a black static color? Does it have an impact on dark mode?

Reply:
The play/pause button, retains its white background and black icon color even on dark mode. We are using the `staticBlackColorText` for the color of the icon.

![Screenshot 2023-05-09 at 2 59 26 PM](https://github.com/GetStream/stream-chat-swift/assets/472467/c851f1cb-f50e-4e3a-807b-f5b1ad6bffc4)